docs/source/model_design/concept_model/standard_results_formats/index.rst

```diff
@@ -0,0 +1,197 @@
+
+2. Successful execution of these standard formats depends on researcher documentation of requested model outputs within the simulation concept model document.
+
+.. todo::
+
+  Link to updated template doc when ready
+
+3. If a researcher-requested output does not clearly fit within the examples outlined below, researchers and engineers should meet to determine desired results formatting and document the format within the project concept model requested outputs section.
+
+Parameters in results dataframes
+++++++++++++++++++++++++++++++++
+
+.. list-table:: Parameters
+  :header-rows: 1
+
+  * - Parameter
+    - Value type
+    - Definition
+    - Possible values
+  * - :code:`input_draw`
+    - integer
+    - Simulated input draw number
+    - [0, 999]
+  * - :code:`scenario`
+    - string
+    - Simulated scenario name
+    - * baseline
+      * alternative
+      * etc.
+  * - :code:`age`
+    - string
+    - Age group of stratum
+    - :code:`gbd_age_group_name.replace(" ","_").lower()`
+      
+      * early_neonatal
+      * 50_to_54
+      * etc.
+  * - :code:`sex`
+    - string
+    - Sex of stratum
+    - * male
+      * female
+  * - :code:`year`
+    - integer
+    - Simulated year
+    - * 2022 
+      * 2023
+      * etc.
+  * - :code:`{other_stratifying_variables}` (example: :code:`treatment`)
+    - string
+    - Some other variable by which to stratify values (in the same way as age, sex, year, etc.). Intervention coverage is a common example.
+    - Name and values of {other_stratifying_variables} to be defined in advance of implementation by researchers.
+
+
+      Example:
+
+      * covered
+      * uncovered
+  * - :code:`measure`
+    - string
+    - Description of what is being measured about the entity
+    - * ylds
+      * ylls
+      * deaths
+      * person_time
+      * transition_count
+      * first_moment
+      * second_moment
+  * - :code:`entity_type`
+    - string
+    - Type of entity being measured
+    - * cause
+      * rei
+      * {other}
+  * - :code:`entity`
+    - string
+    - Name of entity being measured
+    - :code:`gbd_{entity_type}_name.replace(" ","_").lower()`
+      
+      * measles
+      * child_stunting
+  * - :code:`sub_entity`
+    - string
+    - Name of entity state or category
+    - * measles/susceptible_to_measles
+      * cat1/cat2/cat3/cat4
+  * - :code:`value`
+    - float
+    - Measured value
+    - (-inf, inf)
+
+.. note::
+
+  As a reminder, the definitions of the first and second moments for a continuous exposure measure, :math:`Y`, for our purposes are defined below. 
+
+  .. math::
+
+    \text{First moment} = \sum_{i}Y_i \times \text{person time}_i
+
+    \text{Second moment} = \sum_{i}Y_i \times \text{person time}_i
```

Comment:
This probably isn't supposed to be the same as the first?

Reply:
Good catch! Yes, it's updated now.